src/player.rs

```diff
@@ -11,6 +13,18 @@ use metadata::{FileFormat, Track, TrackRef};
 use session::{Bitrate, Session};
 use util::{self, ReadSeek, SpotifyId, Subfile};
 pub use spirc::PlayStatus;
+use std::sync::atomic::{AtomicUsize, Ordering};
+use std::process::exit;
+
+use nix::sys::signal;
+
+static mut received_signal: AtomicUsize = AtomicUsize::new(0);
```

Comment:
Agreed. I will replace it. I would like to add support for handling more signals, thats why an integer.